aws/table_aws_wellarchitected_lens_review_report.go

```diff
@@ -0,0 +1,133 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/wellarchitected"
+	"github.com/aws/aws-sdk-go-v2/service/wellarchitected/types"
+
+	wellarchitectedv1 "github.com/aws/aws-sdk-go/service/wellarchitected"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsWellArchitectedLensReviewReport(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_wellarchitected_lens_review_report",
+		Description: "AWS Well-Architected Lens Review Report",
+		List: &plugin.ListConfig{
+			ParentHydrate: listWellArchitectedWorkloads,
+			Hydrate:       getWellArchitectedLensReviewReports,
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"ResourceNotFoundException"}),
+			},
```

Comment:
I think we need to handle the `ResourceNotFoundException` separately like https://github.com/turbot/steampipe-plugin-aws/blob/main/aws/table_aws_wellarchitected_answer.go#L234-L239

Reply:
In this table we need not to handle the `ResourceNotFoundException` separately like https://github.com/turbot/steampipe-plugin-aws/blob/main/aws/table_aws_wellarchitected_answer.go#L234-L239 because we are not passing any user input which has been passed in where clause, we are just continuing the loop if there is a any mismatch between the user input and the hydrated date (https://github.com/turbot/steampipe-plugin-aws/pull/1697/files#diff-938b6046628f5426df8e95842df2373d36b30c142a9df2513ffb83e64e88ef5bR97). Which will restrict the API call if there is any mismatch, so the list call will never throw `ResourceNotFoundException`.